eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/SummaryStatistics.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * A Summarizer can be used to aggregate statistics for multiple primitive attributes.
+ *
+ * @since 8.1
  */
 public class SummaryStatistics<T> implements Procedure<T>
```

Comment:
I didn't make SummaryStatistics serializable because Int, Long, DoubleSummaryStatistics classes are not serializable.  I'll optimize imports on both of the classes you indicated.  Thank you.